alea/utils.py

```diff
@@ -10,15 +10,18 @@ def get_analysis_space(analysis_space: dict) -> list:
 
     for element in analysis_space:
         for key, value in element.items():
-            if value.startswith("np."):
+            if isinstance(value, str) and value.startswith("np."):
```

Comment:
Do you have an example for usage of this?

Reply:
The example config file has the analysis space defined in the following way:
```python
analysis_space:
        - "cs1": 'np.arange(0, 102, 2)'
        - "cs2": 'np.geomspace(100, 100000, 51)'
```
This will be evaluated accordingly and saves a lot of space compared to the implementation in binference. However, I also re-enabled the binference-like definition, which can be useful when dealing with irregular binnings. To test this you could replace the definition of cs1 with the following:
```python
- "cs1": "0. 2.   4.   6.   8.  10.  12.  14.  16.  18.  20.  22.  24.  26.  28.
  30.  32.  34.  36.  38.  40.  42.  44.  46.  48.  50.  52.  54.  56.
  58.  60.  62.  64.  66.  68.  70.  72.  74.  76.  78.  80.  82.  84.
  86.  88.  90.  92.  94.  96.  98. 100."

```
and it should do exactly the same thing.

Reply:
Also, the example I posted in slack features both implementations (irregular binning in R).